index.js

```diff
@@ -374,13 +405,24 @@ Filter.prototype.canProcessFile =
 
 Filter.prototype.isDirectory = function(relativePath, entry) {
   // @ts-ignore
-  if (this.inputPaths === undefined) {
+  if (this.fsMergerFiles === undefined && this.inputPaths === undefined) {
     return false;
   }
-
-  // @ts-ignore
-  let srcDir = this.inputPaths[0];
-  let path = srcDir + '/' + relativePath;
+  let path = '';
+  if (this.fsMergerFiles) {
```

Comment:
The reason the initialization of `this.fsMergerFiles` because `broccoliNode.outputPath` is available only in the build step, which is needed for fsMerger to work effectively.